### ISSUE TITLE
Tree keyboard navigation bug

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#1356]
this doesn’t work:

tab into tree, tab out, tab back in, click a new item
it doesn’t select the item
